CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -618,8 +631,18 @@ protected virtual bool TrySetProperty(long objectId, string name, object value,
             }
             try
             {
+#if NETCOREAPP
                 property.SetValue(obj.Value, value);
```

Comment:
Same here. We should be able to exclude the whole method for simplicity.

Reply:
@amaitland Excluded the TrySetPropperty for .Net core App.